packages/docs/PropTables/TablePropTable.tsx

```diff
@@ -192,6 +201,27 @@ const tableSortableProps: Prop[] = [
   },
 ];
 
+const tableSearchProps: Prop[] = [
+  {
+    name: 'value',
+    types: 'string',
+    description: 'Value of input',
+    required: true,
+  },
+  {
+    name: 'onSearchChange',
```

Comment:
What are your thoughts on simplifying this API further? Maybe to just `onChange`?

Reply:
I can agree with it. We use this props only in the `Search` component and it would be a bit more simply 
I will change it